File: Source/ArrayRelatedOperations/mean.c
#include <stdio.h>

double calculateMean(int arr[], int n) {
    int sum = 0;
    for (int i = 0; i < n; i++) {
        sum += arr[i];
    }
    return (double)sum / n;
}

int main() {
    int arr[] = {1, 2, 3, 4, 5};
    int n = sizeof(arr) / sizeof(arr[0]);
    printf("Mean = %.2f", calculateMean(arr, n));
    return 0;
}

File: Source/ArrayRelatedOperations/mean.js
function calculateMean(arr) {
    var sum = 0;
    for (var i = 0; i < arr.length; i++) {
        sum += arr[i];
    }
    return sum / arr.length;
}

var arr = [1, 2, 3, 4, 5];
console.log("Mean = " + calculateMean(arr));
